index.js

```diff
@@ -26,19 +30,19 @@ require('fs').access(program.image, function(err){
   if(err) exit("Can't read file.")
 });
 
-// User must input a step.
-if(!program.step || !sequencer.modulesInfo().hasOwnProperty(program.step))
+// User must input a step. If steps exist, check that every step is a valid step.
+if(!program.step || !program.step.every(function(step){return sequencer.modulesInfo().hasOwnProperty(step)}))
```

Comment:
Sorry for being choosy here, but could you make this a bit more legible? Thanks!

Reply:
Should I abstract that line to a function?